### ISSUE TITLE
Deprecated build options

### ISSUE THREAD

**Issue by Penaz91** [Vinetos/Tranquille#8]
While messing around with the repository, I noticed gradle complaining about some deprecated options. I'm leaving the message here as a memo:

```
> Configure project :app
WARNING: The option setting 'android.defaults.buildfeatures.buildconfig=true' is deprecated.
The current default is 'false'.
It will be removed in version 9.0 of the Android Gradle plugin.
You can resolve this warning in Android Studio via `Refactor` > `Migrate BuildConfig to Gradle Build Files`
```

Thank you for taking the time to work on this project!

**Comment by Vinetos** [comment]
Hello ! 
I saw this message too, I think it has been created when I had started to migration to Gradle
I will try to have a look some day after #6

**Comment by Vinetos** [comment]
Closed by #9
